Content.Server/ParticleAccelerator/EntitySystems/ParticleAcceleratorSystem.ControlBox.cs

```diff
@@ -76,9 +81,12 @@ public void SwitchOn(EntityUid uid, ICommonSession? user = null, ParticleAcceler
 
         if (comp.Enabled || !comp.CanBeEnabled)
             return;
+        if (!(user?.AttachedEntity is { }))
```

Comment:
`if (user?.AttachedEntity is not {} player)`

then use player instead of user.AttachedEntity.Value